test/golomb-test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const assert = require('bsert');
+const Block = require('../lib/primitives/block');
+const Opcode = require('../lib/script/opcode');
+const Script = require('../lib/script/script');
+const Golomb = require('../lib/golomb/golomb');
+const testVector = require('./data/filter-valid.json').slice(2);
+
+describe('Golomb test', function() {
+  it('should match all valid input & output scripts', function() {
+    const OP_RETURN = Opcode.fromSymbol('return');
+
+    for (const testCase of testVector) {
+      // Building an array of raw items from the test vector data
+      const items = []; // Corresponds to the L vector in the BIP-158
+      const [, hash, blk, prevScripts, , filter] = testCase;
+      const block = Block.fromRaw(Buffer.from(blk, 'hex'));
+      if (prevScripts.length > 0) {
+        const scripts = prevScripts.filter(str => str.length > 0).map(str => Script.fromRaw(str, 'hex'));
+        // Adding previous output scripts
+        items.push(...scripts);
+      }
+      block.txs.forEach((tx) => {
+        tx.outputs.forEach((out) => {
+          // Only adding output script of length > 0 and which don't contain OP_RETURN
+          if (out.script.length > 0 && !out.script.toArray().find(element => OP_RETURN.equals(element))) {
```

Comment:
nit: Line is too long (max of 80 columns), and single line `if` omit the brackets throughout the code base.

Reply:
How about I replace it by this:

```javascript
const {script} = out;
const hasOpReturn = script.toArray()
                        .find(element => OP_RETURN.equals(element));
// Only adding scripts of length > 0 and which don't contain OP_RETURN
if (script.length > 0 && !hasOpReturn)
    items.push(out.script);
```
Obs
* The brackets around the `if` were removed
* By defining the `hasOpReturn` constant, the `if` condition is more clearly understandable (maybe even to the point of not requiring the comment)
* The method chaining used to generate the `hasOpReturn` constant makes the line exceed the 80 column mark, so I'm breaking it like that. Is that acceptable?